ctlearn/data_loading.py

```diff
@@ -9,18 +9,32 @@
 import tables
 
 from ctlearn.data_processing import DataProcessor
-from ctlearn.image_mapping import ImageMapper
+from ctlearn.image_mapping import ImageMapper, get_camera_type
+
+# TEMPORARY: Maps old telescope names into new
+# For training with datasets generated with versions prior to image-extractor v0.6.0
+OLD_TEL_NAMES_TO_NEW = {
```

Comment:
These dicts are defined twice, in data_loading and image_mapping. Maybe define them just in image_mapping and import them.

Reply:
Removed in c69382f22114aacdaba257c89f0b8d96ea334b0e.